libtsuba/include/katana/TxnContext.h

```diff
@@ -85,9 +92,11 @@ class KATANA_EXPORT TxnContext {
 
   bool GetTopologyWrite() const { return topology_write_; }
 
+  bool GetCommitManifest() const { return commit_manifest_; }
+
```

Comment:
I tend to agree with @Amber that `Get `and `Set `are names of last resort.  I think `CommitManifest()` would be a better name.

Reply:
I see. I'll change to remove all the `Get`-s. But what did he suggest to deal with `Set`?